src/python/pants/backend/python/util_rules/pex_environment.py

```diff
@@ -29,6 +29,23 @@ class PexRuntimeEnvironment(Subsystem):
     options_scope = "pex"
     help = "How Pants uses Pex to run Python subprocesses."
 
+    run_packaged_firstparty_code = BoolOption(
```

Comment:
I need to issue an error if the user doesn't set this. I assume the way to do that is make the classvar "private", then expose a property with this name, and in the property check `if self.options.is_default("run_packaged_firstparty_code")` and error.

That sound right @Eric-Arellano ?

Reply:
Nevermind. implemented.